src/main/proto/com/google/api/codegen/config.proto

```diff
@@ -333,9 +333,11 @@ message SampleValueSet {
   //   several placeholders.
   repeated string parameters = 4;
 
+  repeated string sample_args = 5;
```

Comment:
I'd ask for a comment, but we agreed on a new schema that you'll implement in a chaser PR. :-).

However, I would just add a very temporary TODO saying that this will go away in favor of a new schema shortly, just to avoid confusion for folks looking at it in the interim.

Reply:
Done